src/renderers/shaders/UniformsUtils.js

```diff
@@ -73,6 +75,19 @@ export function cloneUniformsGroups( src ) {
 
 }
 
+export function getUnlitUniformColorSpace( renderer ) {
+
+	if ( renderer.getRenderTarget() === null ) {
+
+		// https://github.com/mrdoob/three.js/pull/23937#issuecomment-1111067398
+		return renderer.outputEncoding === sRGBEncoding ? SRGBColorSpace : LinearSRGBColorSpace;
+
+	}
+
+	return LinearSRGBColorSpace;
```

Comment:
Shouldn't this return the color space of the render target?

Reply:
This is explained here: https://github.com/mrdoob/three.js/pull/23937#issuecomment-1111067398